packages/amplify-cli/src/extensions/amplify-helpers/resource-status.js

```diff
@@ -56,7 +56,30 @@ function capitalize(str) {
   return str.charAt(0).toUpperCase() + str.slice(1).toLowerCase();
 }
 
-function getAllResources(amplifyMeta, category, resourceName) {
+function getCommonResources(resources, filteredResources) {
```

Comment:
Can we name this differently. Some thing like `filterResources` or something as getCommonResource doesn't convey the meaning